raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java

```diff
@@ -598,14 +600,6 @@ private boolean hasInconsistentLeader(int epoch, OptionalInt leaderId) {
         }
     }
 
-    private void becomeFollower(int epoch, OptionalInt leaderId) throws IOException {
```

Comment:
I eventually decide to remove this call because it has a very similar appearance towards `becomeFollower(leaderId, epoch)` and as it is only called once.